code-challenges.js

```diff
@@ -1,32 +1,58 @@
 // ASSESSMENT 6: JavaScript Coding Practical Questions
 
-
 // --------------------1) Create a function that takes in an array of objects and returns a sentence about each person with their names capitalized.
 
 var people = [
-  { name: "ford prefect", occupation: "hitchhiker" },
-  { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
-  { name: "arthur dent", occupation: "radio employee" }
-]
-// Expected output example: "Ford Prefect is a hitchhiker." "Zaphod Beeblebrox is a president of the galaxy." "Arthus Dent is a radio employee."
-
-
-
+	{ name: "ford prefect", occupation: "hitchhiker" },
+	{ name: "zaphod beeblebrox", occupation: "president of the galaxy" },
+	{ name: "arthur dent", occupation: "radio employee" },
+];
+
+const sentence = (array) => {
+  //initially map through array to obtain values within
+  let firstLetterCap = array.map(value => {
+    //split the values(names) within people.name
+    let splitNames = value.name.split(" ")
+      //people.name map to capitalize only first char for both names + interpolate people.occupation
+      return splitNames.map(value => value.charAt(0).toUpperCase() + value.substring(1)).join(" ") +  is a ${value.occupation}.
```

Comment:
Great job splitting up the names and using you're charAt to hone in on just the first letter of their names to join them back again with str interpolation. Another way I found to answer this was to use: splitNames.map (val=> val[0].toUpperCase() + val.slice(1)).join(" ")